### PR TITLE
Use OWL also for mono-coil reconstruction

### PR DESCRIPTION
**Pull request by paquiteau**
Hi, small (but first ;) ) PR to make OWL regularization also possible for single coil reconstruction.  I also documented the `scale_based` option.

### CODE REVIEW

**Comment by chaithyagr** [comment]
Wait, i am a bit confused, why would you need OWL regularization for single coil? I mean where are you enforcing the group sparsity with ?

File: mri/operators/proximity/ordered_weighted_l1_norm.py [hunk 1]
```diff
@@ -27,11 +27,13 @@ class OWL(ProximityParent):
...
     n_coils: int
         number of channels
-    mode: string 'all' | 'band_based' | 'coeff_based', default 'band_based'
+    mode: string 'all' | 'band_based' | 'coeff_based' | 'scale_based',
+        default 'band_based'
         Mode of operation of proximity:
         all         -> on all coefficients in all channels
         band_based  -> on all coefficients in each band
         coeff_based -> on all coefficients but across each channel
+        scale_based -> on al coefficients in each scale
```
Comment (chaithyagr) [inline mri/operators/proximity/ordered_weighted_l1_norm.py:36]:
```suggestion
        scale_based -> on all coefficients in each scale
```

**Comment by paquiteau** [comment]
The group spasity is not taken into account, but I wanted to try the regularisation per band/scale. OWL could also be implemented (in a not so near future) on the "growing tree" of coefficient for a specific band , across scales.

**Comment by chaithyagr** [comment]
@paquiteau is this ready for review? I am sorry I have not seen it yet.

**Comment by chaithyagr** [comment]
Particularly, you have some code style PEP issues:

```
$ pycodestyle mri --ignore="E121,E123,E126,E226,E24,E704,E402,E731,E722,E741,W503,W504,W605"
mri/operators/proximity/ordered_weighted_l1_norm.py:117:23: E231 missing whitespace after ','
mri/operators/proximity/ordered_weighted_l1_norm.py:118:29: E221 multiple spaces before operator
The command "pycodestyle mri --ignore="E121,E123,E126,E226,E24,E704,E402,E731,E722,E741,W503,W504,W605"" exited with 1.
```

Can you please ensure they work and pass?

**Comment by chaithyagr** [comment]
CLosing as it is stale...

**Pull request closed without merging**